src/Shared/@components/Input/Input.module.scss

```diff
@@ -0,0 +1,33 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px 24px;
+  width: 456px;
+  background-color: #ffffff;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.1);
```

Comment:
No-no, why do we need a card on which you place the input? We don't. Input component only styles INPUT, this lil thing
![image](https://user-images.githubusercontent.com/52495853/102395669-2291b580-3fe4-11eb-8002-3609768d589b.png)

Reply:
Yes, sempai!